Added language es-es

zeedif opened the pull request.

zeedif committed:
Added language es-es

Tictim commented on src/main/resources/assets/paraglider/lang/es_es.json:
```diff
@@ -0,0 +1,16 @@
+	"tooltip.heart_container.1": "Aumenta la salud máxima en %s, hasta %s veces.",
+	"tooltip.heart_container.1.hearts": "1 corazón",
+	"tooltip.stamina_vessel.1": "Aumenta la resistencia máxima en %s, hasta %s veces.",
+	"tooltip.horned_statue.1": "Recupera todos los contenedor de corazón y de vigor consumidos.",
+	"tooltip.horned_statue.2": "¡PRECIO ESPECIAL! ¡No se requieren rupias! Adorable como el infierno."
```
These two lines were removed. See [here](https://github.com/Tictim/Paraglider/blob/release/1.16/src/main/resources/assets/paraglider/lang/en_us.json#L26).

Tictim reviewed (changes requested):
First of all, thanks for the contribution! But unfortunately the lang file seems outdated. The text grew tenfold with the new updates, so if you don't want to subject yourself then it's fine. But even with that case, I want to request an update, because the update makes some of the text straight up a lie :/

Or, if you want to translate updated text, then you can find it in [here](https://github.com/Tictim/Paraglider/blob/release/1.16/src/main/resources/assets/paraglider/lang/en_us.json). New beta build that will be released soon(tm) will also contain this. A few notes for anyone willing to translate:

* Name of the advancements are original except Statue's Bargain, which is name of the quest in BotW.
* Almost all bargain dialogs are snippets from BotW. Matching translated text to the BotW translation is preferred.
* Not all of the dialogs are from BotW, though; like `bargain.dialog.horned_statue.failure.essence_full.0` which is original.
* TODOs in bargain dialogs means not yet written. These are fallback text for bargains that do not possess any of set criteria to define what failure dialog should be put.
  For example, Goddess Statue has special failure dialog for case of missing items, full heart and full stamina. Recipes added by datapacks that produces failure reasons outside these will produce fallback dialog. Yeah. I'm terrible explainer. I'll just shut up.

zeedif committed:
Update es_es.json

I need to translate some lines but I don't have the game and I watched some videos to translate these.

Tictim commented:
Seems fine to me. I will cull out only translated strings before merging. You may want to look out for beta build to see the text in game and check everything sounds right. Again, thanks for your hard work.
